clang/lib/3C/ConstraintBuilder.cpp

```diff
@@ -104,7 +104,7 @@ class FunctionVisitor : public RecursiveASTVisitor<FunctionVisitor> {
     for (const auto &D : S->decls()) {
       if (VarDecl *VD = dyn_cast<VarDecl>(D)) {
         Expr *InitE = VD->getInit();
-        CB.constrainLocalAssign(S, VD, InitE);
+        CB.constrainLocalAssign(S, VD, InitE, Same_to_Same);
```

Comment:
Isn't `constrainLocalAssign` using `Same_to_Same` by default? Why does this need to be added?